LabExT/Tests/Fixtures/MCSControlInterface.py

```diff
@@ -0,0 +1,601 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+LabExT  Copyright (C) 2022  ETH Zurich and Polariton Technologies AG
+This program is free software and comes with ABSOLUTELY NO WARRANTY; for details see LICENSE file.
```

Comment:
Not sure if this is correct. Most of these constants are supplied by SmarAct which have a proprietary license on their drivers.

We can argue okay, we are not supplying the drivers but merely interface with them. But I would add an acknowledgment to SmarAct here.

Reply:
I agree, I wasn't sure if we were allowed to do that. If it's because of the constants, we can also remove them from the interface. Otherwise it is useful to know during testing which methods the `MCSControl_PythonWrapper` module provides. 😄

Reply:
eh... lets keep it for now, Ill ask SmarAct via Email if this is okay.